static/app/views/performance/newTraceDetails/traceWaterfall.tsx

```diff
@@ -524,6 +524,21 @@ export function TraceWaterfall(props: TraceWaterfallProps) {
         action_source: 'load',
       });
     });
+
+    // TODO Abdullah Khan: Remove this once /trace-meta/ starts responding
+    // with the correct spans count for EAP traces.
+    if (
+      traceNode &&
+      isEAPTraceNode(traceNode) &&
+      props.tree.eap_spans_count !== props.meta?.data?.span_count
+    ) {
+      Sentry.withScope(scope => {
+        scope.setFingerprint(['trace-eap-spans-count-mismatch']);
```

Comment:
is it necessary to manually set a fingerprint here? won't this still be grouped based on the LOC?

Reply:
Seems like most places logging an event like this sets the finger print, I just took inspo. Should I remove it @DominikB2014 ?